lib/pdfkit/pdfkit.rb

```diff
@@ -80,6 +88,11 @@ def to_file(path)
     File.new(path)
   end
 
+  def is_windows?
+    require 'rbconfig'
```

Comment:
We're going to call this method possibly many times each time we build up a command. Let's just put this require at the top of the file with the require of shellwords.